src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -53,29 +53,23 @@ CA1068 | Design | Info | CancellationTokenParametersMustComeLastAnalyzer, [Docum
 CA1069 | Design | Info | EnumShouldNotHaveDuplicatedValues, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1069)
 CA1070 | Design | Info | DoNotDeclareEventFieldsAsVirtual, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1070)
 CA1200 | Documentation | Hidden | AvoidUsingCrefTagsWithAPrefixAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1200)
-CA1300 | Globalization | Disabled | SpecifyMessageBoxOptionsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1300)
-CA1301 | Globalization | Disabled | AvoidDuplicateAcceleratorsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1301)
 CA1303 | Globalization | Disabled | DoNotPassLiteralsAsLocalizedParameters, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1303)
 CA1304 | Globalization | Hidden | SpecifyCultureInfoAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1304)
 CA1305 | Globalization | Hidden | SpecifyIFormatProviderAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1305)
-CA1306 | Globalization | Disabled | SetLocaleForDataTypesAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1306)
 CA1307 | Globalization | Disabled | SpecifyStringComparisonAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1307)
 CA1308 | Globalization | Disabled | NormalizeStringsToUppercaseAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1308)
 CA1309 | Globalization | Hidden | UseOrdinalStringComparisonAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1309)
 CA1310 | Globalization | Hidden | SpecifyStringComparisonAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1310)
 CA1401 | Interoperability | Info | PInvokeDiagnosticAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1401)
```

Comment:
Unrelated to the PR:
This analyzer name ends with "DiagnosticAnalyzer" while others ends in just "Analyzer". Do you want to keep consistency in the naming, or it doesn't matter for you?

Reply:
I don't think we care much about analyzer names, at least we haven't in this repo.